Datadog/Example/Scenarios/RUM/RUMScenarios.swift

```diff
@@ -74,6 +74,38 @@ final class RUMModalViewsAutoInstrumentationScenario: TestScenario {
     private class Predicate: UIKitRUMViewsPredicate {
         func rumView(for viewController: UIViewController) -> RUMView? {
             if let viewName = viewController.accessibilityLabel {
+                print(viewName)
+                return .init(name: viewName)
+            } else {
+                return nil
+            }
+        }
+    }
+
+    func configureSDK(builder: Datadog.Configuration.Builder) {
+        _ = builder
+            .trackUIKitRUMViews(using: Predicate())
+            .enableLogging(false)
+            .enableTracing(false)
```

Comment:
Side note:

Would be great to improve the API in V2. IMO it should be `.loggingEnabled(true)` an so on. cc @maxep